dacbench/wrappers/observation_wrapper.py

```diff
@@ -22,7 +22,6 @@ def __init__(self, env):
         super(ObservationWrapper, self).__init__(env)
         obs_sample = self.flatten(self.env.observation_space.sample())
         size = len(obs_sample)
-        print(obs_sample)
```

Comment:
It seems I forgot to remove this one from the commit history. Is this print really necesarry?